packages/web-app-files/src/components/Spaces/QuotaModal.vue

```diff
@@ -185,28 +197,35 @@ export default {
     setOptions() {
       this.options = [...this.DEFAULT_OPTIONS]
 
-      const selectedQuotaInOptions = this.options.find(
-        (option) => option.value === this.space.spaceQuota.total
-      )
+      let selectedQuotaInOptions = null
+
+      if ('total' in this.space.spaceQuota) {
+        selectedQuotaInOptions = this.options.find(
+          (option) => option.value === this.space.spaceQuota.total
+        )
+      } else {
+        selectedQuotaInOptions = this.options.find((option) => option.unlimited === true)
+      }
 
       if (selectedQuotaInOptions) {
         this.selectedOption = selectedQuotaInOptions
-      } else {
-        const newOption = {
-          displayValue: (this.space.spaceQuota.total * Math.pow(10, -9))
-            .toFixed(2)
-            .toString()
-            .replace('.00', ''),
-          displayUnit: 'GB',
-          value: this.space.spaceQuota.total
-        }
-        this.options.push(newOption)
-        this.options = [
-          ...this.options.filter((o) => o.value).sort((a, b) => a.value - b.value),
-          ...this.options.filter((o) => !o.value)
-        ]
-        this.selectedOption = newOption
+        return
       }
+
+      const newOption = {
+        displayValue: (this.space.spaceQuota.total * Math.pow(10, -9))
+          .toFixed(2)
```

Comment:
Would maybe extract this into a variable for better readability, but not a show stopper.

Reply:
true